drivers/backup/portworx/portworx.go

```diff
@@ -390,7 +391,16 @@ func (p *portworx) GetVolumeBackupIDs(
 		}
 		logrus.Debugf("GetVolumeBackupIDs storkApplicationBackupCR: [%+v]\n", storkApplicationBackupCR)
 		if len(storkApplicationBackupCR.Status.Volumes) > 0 {
-			return false, false, nil
+			isComplete := true
+			for _, backupVolume := range storkApplicationBackupCR.Status.Volumes {
+				logrus.Debugf("Volume [%v] has backup ID: [%v]\n", backupVolume.Volume, backupVolume.BackupID)
+				if !strings.Contains(backupVolume.BackupID, "/") {
```

Comment:
why checking for a slash?

Reply:
This particular test restarts px service on the node from where cloud snap backup is in-progress. To find out this node,
1. Test inspects px-backup backup object to get correct stork applicationBackup CR.
2. This stork CR contains the px cloud-snap ID.
3. PX cloud-snap with above id is inspected to find out on which node it is running.
 
In recent px-backup torpedo [runs](https://jenkins.portworx.co/job/Torpedo/view/Torpedo%20NextPX/job/tp-nextpx-backup/148/console) it has been seen that some time stork CR is getting populated with only the last segment of cloud-snap id. (For e.g. just `612306637010542309` instead of complete `tp-backup-bucket-07-10-13h50m50s/1052617247788136551-612306637010542309`). This CR is updated by stork from time to time with correct details. So, have made this changes so that test waits for complete cloud-snap id to get populated

Reply:
@thiguetta can u pls take a look at this?